restcomm/restcomm.ui/src/main/webapp/resources/js/controllers/numbers-incoming.js

```diff
@@ -70,6 +70,7 @@ rcMod.controller('NumberDetailsCtrl', function ($scope, $stateParams, $location,
     $scope.localVoiceApps = Applications.filterByKind(localApps, 'voice');
     $scope.localSmsApps = Applications.filterByKind(localApps, 'sms');
     $scope.localUssdApps = Applications.filterByKind(localApps, 'ussd');
+    $scope.localReferApps = Applications.filterByKind(localApps, 'voice');
```

Comment:
$scope.localReferApps can be removed if localVoiceApps are used in numbers-incoming-details.html